collections/bff/delegation/Approve Delegation.bru

```diff
@@ -0,0 +1,26 @@
+meta {
+  name: Approve delegation
+  type: http
+  seq: 1
+}
+
+post {
+  url: {{host-bff}}/producer/delegations/:delegationId/approve
+  body: none
+  auth: none
+}
+
+params:path {
+  delegationId: dd10132c-f3c9-45cf-994c-3a312bc4ab4e
+}
+
+headers {
+  Cache-Control: no-cache
+  Content-Type: application/json
+  Accept: */*
+  Accept-Encoding: gzip, deflate, br
+  Connection: keep-alive
+  Authorization: {{JWT}}
+  X-Correlation-Id: {{correlation-id}}
+  Content-Type: application/json
+}
```

Comment:
Could you remove all unneeded headers from all Bruno requests? See other requests as a reference: they should all need just the following

```
headers {
  Authorization: {{JWT}}
  x-correlation-id: {{correlation-id}}
}
```